src/test/crimson/seastore/test_seastore.cc

```diff
@@ -1280,8 +1280,13 @@ TEST_P(seastore_test_t, zero)
 INSTANTIATE_TEST_SUITE_P(
   seastore_test,
   seastore_test_t,
-  ::testing::Values (
-    "segmented",
-    "circularbounded"
+  ::testing::Combine(
+    ::testing::Values (
+      "segmented",
+      "circularbounded"
+    ),
+    ::testing::Values(
+      integrity_check_t::FULL_CHECK,
+      integrity_check_t::NONFULL_CHECK)
```

Comment:
Make check is failed by:
261 - unittest-seastore (Timeout)

Probably it doesn't worth to repeat all the tests for FULL vs NONFULL_CHECK ?

Same applies to other tests.

Reply:
@cyx1231st Seems that the timeout of "unittest-seastore" is not caused by this PR, #56998 encountered the same problem

Reply:
> Seems that the timeout of "unittest-seastore" is not caused by this PR, https://github.com/ceph/ceph/pull/56998 encountered the same problem

Thanks for the info!

> 282/293 Test #261: unittest-seastore .........................   Passed  522.94 sec
> 283/293 Test #258: unittest-object-data-handler ..............   Passed  651.48 sec
> 292/293 Test #254: unittest-transaction-manager ..............   Passed  2335.45 sec

I'm still worried that the test time is already long and expensive. Should we select one test case for each related UT in order to validate FULL vs NONFULL, what do you think?

Reply:
Yeah, will try.

Reply:
> 276/293 Test #261: unittest-seastore .........................   Passed  382.67 sec
> 289/293 Test #254: unittest-transaction-manager ..............   Passed  913.35 sec

The costs look much better.